install/kubernetes/helm/istio/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: istio
-version: 1.1.0
+version: 1.1.1
```

Comment:
the next istio release is 1.1.0 - please revert this change

Reply:
OK. But how do you track changes on the chart?

Reply:
Per release of Istio, Test and Relaese (T&R) seds this file to the next major version (1.2.0).  We don't track individual changes -1,-2, etc as Helm doesn't have any mechanism of templating version fields.  As we have a slew of version fields, tracking individual changes is not tenable.